modules/controller/landing_pack_tracking.py

```diff
@@ -0,0 +1,91 @@
+"""
+keeps track of detected and visited landing pads
+"""
+
+import numpy as np
+
+from .. import object_in_world  #Edit name when finalized
+
+
+class LandingPadTracking:
+    """
+    keeps track of the real world location of landing pads that
+    are to be visited or have already been vsisited
+    """
+    def __init__(self, distance_threshold: float):
+        self.__unconfirmed_positives = []
+        self.__false_positives = []
+        self.__confirmed_positives = []
+
+        # Landing pads within the square root of this distance are considered the same landing pad
+        self.__distance_threshold = distance_threshold
```

Comment:
Comment and variable name is contradictory, does distance_threshold represent distance or distance^2? If it represents distance^2 then the name should reflect that.